api_tester/lib/api_tests/models/package_wrapper_api_test.dart

```diff
@@ -26,16 +27,21 @@ class _PackageApiTest {
     Map<String, dynamic> json = package.toJson();
   }
 
-  void _checkConstructor(String identifier, PackageType packageType,
-      StoreProduct storeProduct, String offeringIdentifier) {
-    Package package =
-        Package(identifier, packageType, storeProduct, offeringIdentifier);
+  void _checkConstructor(
+      String identifier,
+      PackageType packageType,
+      StoreProduct storeProduct,
+      PresentedOfferingContext presentedOfferingContext) {
+    Package package = Package(
+        identifier, packageType, storeProduct, presentedOfferingContext);
```

Comment:
This replaces `String offeringIdentifier` with `PresentedOfferingContext presentedOfferingContext` in the constructor.  Technically a breaking change but I developer should not be using this one.

Reply:
I was thinking we could potentially create a new constructor for the presentedOfferingContext to avoid the breaking change, but dart is annoying and only lets you have a single unnamed constructor, which should be the old one, making things complex... So I'm ok with this I think.